src/main/cpp/t02_new.cpp

```diff
@@ -25,10 +25,23 @@
 
 #include "t02_new.h"
 #include <iostream>
-
+#include <set>
 
 using namespace std;
 
 int t02_new() {
-    
+	int n, k, s1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 's1' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4676905)